src/frequenz/sdk/actor/power_distributing/_component_managers/_battery_manager.py

```diff
@@ -668,6 +672,14 @@ def _parse_result(
                     battery_ids,
                     request_timeout.total_seconds(),
                 )
+            except Exception as e:  # pylint: disable=broad-except
+                failed_power += distribution[inverter_id]
+                failed_batteries = failed_batteries.union(battery_ids)
+                _logger.warning(
+                    "Unknown error while setting power to battery %s: %s",
+                    battery_ids,
+                    e,
+                )
```

Comment:
I recommend using `_logger.exception()` here, if this is an unexpected error (i.e. a bug), it will give us much more context about what and where the problem really happened (all the chained exceptions and a full traceback). If you think the log level should not be error and should be warning instead, you can just pass `exc_info=True, stack_info=True`.
```suggestion
            except Exception:  # pylint: disable=broad-except
                _logger.exception(
                    "Unknown error while setting power to battery %s",
                    battery_ids,
                )
```

Reply:
done